login.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓 - 로그인</title>
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/static/pretendard.css"
+    />
+    <link rel="stylesheet" href="css/styles.css" />
+  </head>
+  <body>
+    <div class="login__wrap">
+      <header>
+        <a href="/" class="header__logo">
+          <img
+            class="header__logo-img"
+            src="images/logo/logo-lg.png"
+            alt="판다마켓 로고"
+          />
+        </a>
+      </header>
+
+      <main>
+        <div class="main__wrap">
+          <form class="login-form">
+            <label for="email" class="login-form__label">이메일</label>
+            <input
+              id="email"
```

Comment:
P3:
라벨이랑 인풋이랑 연결해주신거 좋습니다 👍

```suggestion
            <label for="email" class="login-form__label">이메일</label>
            <input
              id="email"
```